services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/attachment/AttachmentController.kt

```diff
@@ -0,0 +1,234 @@
+package fi.hel.haitaton.hanke.attachment
+
+import fi.hel.haitaton.hanke.HankeNotFoundException
+import fi.hel.haitaton.hanke.HankeService
+import fi.hel.haitaton.hanke.currentUserId
+import fi.hel.haitaton.hanke.permissions.PermissionCode
+import fi.hel.haitaton.hanke.permissions.PermissionCode.EDIT
+import fi.hel.haitaton.hanke.permissions.PermissionCode.VIEW
+import fi.hel.haitaton.hanke.permissions.PermissionService
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.media.Content
+import io.swagger.v3.oas.annotations.media.Schema
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.responses.ApiResponses
+import java.net.URLConnection
+import java.util.UUID
+import javax.servlet.http.HttpServletResponse
+import mu.KotlinLogging
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.HttpHeaders
+import org.springframework.http.HttpHeaders.CONTENT_DISPOSITION
+import org.springframework.http.MediaType
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.DeleteMapping
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+import org.springframework.web.multipart.MultipartFile
+
+private val logger = KotlinLogging.logger {}
+
+@RestController
+@RequestMapping
+class AttachmentController(
+    @Autowired private val attachmentService: AttachmentService,
+    @Autowired private val hankeService: HankeService,
+    @Autowired private val permissionService: PermissionService,
+) {
+
+    @GetMapping("liite/{id}")
```

Comment:
Why change these to singluar? Isn't plural the usual convention?

Reply:
I guess singular is just my personal preference. I don't think neither is wrong. But plural seems to be the convention here, so I will change this.

Reply:
Fixed.